system/default_ad_api_helpers/README.md

```diff
@@ -0,0 +1,15 @@
+# default_ad_api_helpers
+
+## routing_adaptor
```

Comment:
Will you create another adaptor for `/initialpose`? Or will you add `/initialpose` to this node in the future?

Reply:
The initial_pose_adaptor is in this PR. https://github.com/autowarefoundation/autoware.universe/pull/1431

Reply:
Considering the design, I feel it's more natural to create `ad_api_rviz_adaptor` instead of `routing_adaptor` and `initial_pose_adaptor`.
@isamu-takagi cc @yukkysaito @mitsudome-r What do you think?
![image](https://user-images.githubusercontent.com/31987104/183783244-c5753d38-c2dc-49c4-a5e6-40fc9e64adb5.png)

Reply:
Do you mean merging two functions into one node?

Reply:
Right. Are there any problems?

Reply:
What is the reason for merging? I think it's enough to put them in one package and it's better to have a node for each function.

Reply:
Then, at least, I feel the package should be named like `ad_api_rviz_adaptors`.
`default_ad_api_helpers` and `routing_adaptor` are a bit unclear to me.

I'd like to wait for comments from other developers.

Reply:
Currently, the default_ad_api_helpers nodes are:
- initial_pose_adaptor (convert rviz `initialpose` to `/api/localization/initialize`)
- routing_adaptor (convert rviz `goal` and `checkpoint` to `/api/routing/set_route_points`)
- automatic_pose_initializer (call `/api/localization/initialize` periodically for gnss initialization, not use rviz)

Reply:
Just for your reference, My preference is as follows:
- ad_api_rviz_adaptors (package)
  - initial_pose_adaptor (node)
  - routing_adaptor (node)
  - (Or, everything in one node is fine.)
- automatic_pose_initializer (package)
  - automatic_pose_initializer (node)

Reply:
Can I talk about this at the next meeting? sure, the result will be opened.

Reply:
The result of the discussion is:
- automatic_pose_initializer (pkg)
  - automatic_pose_initializer (node)
- ad_api_adapters (pkg)
  - rviz_routing_adaptor (node)
  - rviz_initial_pose_adaptor (node)

The routing and initial pose adaptor covert the topic to service. They provide similar functionality and are therefore in the same package. The automatic_pose_initializer provides an independent functionality, split it into a separate package.

Reply:
Thank you for considering it! Looks nice. :smile:

Reply:
Moved and renamed the package https://github.com/autowarefoundation/autoware.universe/pull/1494/commits/3d0e560af5e24e3cf42a4b67ee0faa3f1114329d.